designsafe/apps/workspace/models/app_entries.py

```diff
@@ -227,9 +231,12 @@ class AppVariant(models.Model):
     @property
     def href(self):
         """Retrieve the app's URL in the Tools & Applications space"""
-        app_href = f"/rw/workspace/{self.app_id}"
-        if self.version:
-            app_href += f"?appVersion={self.version}"
+        if self.external_href:
+            app_href = self.external_href
+        else:
+            app_href = f"/rw/workspace/{self.app_id}"
+            if self.version:
+                app_href += f"?appVersion={self.version}"
```

Comment:
not necessary, but could be slightly simplified:
```suggestion
        if self.external_href:
            return self.external_href
        
        app_href = f"/rw/workspace/{self.app_id}"
        if self.version:
            app_href += f"?appVersion={self.version}"
```